### DISCUSSION TITLE
Create a new view return error undefined

### DISCUSSION THREAD

**Discussion by techandmedia** [supabase/supabase#14015] [Questions]
So I create a view like this

```
CREATE OR REPLACE VIEW public.v_menu2
AS
WITH grouped_data AS (
    SELECT
        a.item_cat0_industry_name,
        b.item_cat1_kategori_name,
        COUNT(*) AS count
    FROM
        item_cat0_industry a
        LEFT JOIN item_cat1_kategori b ON b.item_cat0_industry_id = a.item_cat0_industry_id
        LEFT JOIN item_cat2_subkategori c ON c.item_cat1_kategori_id = b.item_cat1_kategori_id
        LEFT JOIN item_cat3_jenis d ON d.item_cat2_subkategori_id = c.item_cat2_subkategori_id
    GROUP BY
        a.item_cat0_industry_name,
        b.item_cat1_kategori_name
)
SELECT
    item_cat0_industry_name,
    item_cat1_kategori_name,
    count
FROM
    grouped_data
ORDER BY
    item_cat0_industry_name,
    item_cat1_kategori_name;
```

And when I run the app (NextJs) locally and fetch the data, it return error like this

`GET https://qahrpqxcgrumwkayyowt.supabase.co/rest/v1/v_menu2?select=*&offset=0&limit=10 401`

I am not sure why I got 401 while other table and view are able to fetch without error

I tried to create the view from both the sql editor on supabase or using dbeaver; it return the same error
I tried to change the name of the view but no luck, still error
I tried to use different schema, still error while other table and view from other schema not public works just fine

**Comment by steve-chavez** [comment]
> GET https://qahrpqxcgrumwkayyowt.supabase.co/rest/v1/v_menu2?select=*&offset=0&limit=10 401

Share the full error message(it should have a body) plus how you make the request.
